backend/src/routes/usersFriendsRoute.ts

```diff
@@ -90,13 +90,13 @@ friendshipRouter.get(
 );
 
 friendshipRouter.get(
-  "/:userId/friend-suggestions/:page",
+  "/:userId/friend-suggestions",
   async (req: Request, res: Response) => {
```

Comment:
I don't remember if it was already in code or I added, but changing the Response to custom would cause to change it everywhere where friends are mentioned, like friend-suggestions or friends endpoints, each to corresponing endpoint. I didn't come to any other conslusion, so if you have any, let me know! 😊